Tests/testError.py

```diff
@@ -20,7 +22,50 @@ def test_error(self): #Run viewer.py on all turtle files in data folder - Checks
 			viewer.main(i,i + "test") #Run viewer on turtle file
```

Comment:
Just before calling `viewer.main()` can you delete the output folder so that the tests can be run several times without have the `"directory already exists"` error.